### PR TITLE
[REF][8.0][BR000973][T22092]Refactor elico_project into several distinct modules[project_planning]

### PR DESCRIPTION
**Pull request by dongshukai**
Submit the PR for module project_planning

### CODE REVIEW

File: project_planning/README.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+ * have basic modules installed (project_task_type, project_task_wbs)
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/Elico-Corp/odoo/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/Elico-Corp/odoo/issues/new?body=module:%20project_planning%0Aversion:%20{8.0}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (elicoidal) [inline project_planning/README.rst]: Remove this line following latest template

File: project_planning/README.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+Credits
+=======
+
+Contributors
+------------
+
+* Sébastien Maillard <sebastien.maillard@elico-corp.com>
+* Luke Zheng <luke.zheng@elico-corp.com>
+
+Maintainer
```
Comment (elicoidal) [inline project_planning/README.rst]: Use latest template

File: project_planning/README.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+`here <https://github.com/Elico-Corp/odoo/issues/new?body=module:%20project_planning%0Aversion:%20{8.0}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Sébastien Maillard <sebastien.maillard@elico-corp.com>
+* Luke Zheng <luke.zheng@elico-corp.com>
```
Comment (elicoidal) [inline project_planning/README.rst]: Add my name

File: project_planning/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2015-2017 Elico corp (http://www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Project Planning',
+    'version': '8.0.1.0.3',
```
Comment (elicoidal) [inline project_planning/__openerp__.py]: start with 8.0.1.0.0 as this is the first version of this module

File: project_planning/project.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+# © 2015-2017 Elico corp (http://www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    type = fields.Selection(
```
Comment (elicoidal) [inline project_planning/project.py]: I think this is obsolete
@victormartinelicocorp ?

**Review by elicoidal — CHANGES_REQUESTED** [review]

**Comment by dongshukai** [comment]
The module depend on the #164 and #165 . 
So please merge the #164 and #165 first, and the travis-ci will be green.

**Comment by dongshukai** [comment]
@Elico-Corp/odoo-reviewers 
I have recheck the module elico_project and remove the relaeted task_type and wbs fields. pls check

**Review by elicoidal — APPROVED** [review]

**Pull request merged**